src/solc_0.8/common/BaseWithStorage/WithAdmin.sol

```diff
@@ -3,29 +3,7 @@
 pragma solidity 0.8.2;
 
 contract WithAdmin {
```

Comment:
I would create a new file WithAdminEmpty as we would like to keep using WithAdmin for other contract

Reply:
Yeah, makes sense. I was thinking the same but never made the change.